Update of `pyproject.toml'file; package name changed and entry point removed, after xcube plugins auto-recognition is updated. (#40)
xcube-dev/xcube-cmems: CHANGES.md

## Changes in 0.5.2 (in development)

- Update of `pyproject.toml'file; package name changed and entry point removed, since xcube

>   plugins auto-recognition is updated. (#39 and xcube-dev/xcube#963)

Reviewer: Merging of this PR requires a not-yet-released xcube release. 
So let's not merge this before we know the xcube version that fixes it and name that version here.

> - Update of `pyproject.toml'file; package name changed and entry point removed, since xcube
>   plugins auto-recognition is updated. (#39 and xcube-dev/xcube#963)

Reviewer: ```suggestion
- Updated `pyproject.toml` file; package name changed from `xcube-cmems`
  to `xcube_cmems` and entry point removed, since xcube plugins 
  auto-recognition is updated. (#39 and xcube-dev/xcube#963)
```

Note, line length in `*.md` files should not exceed 80 chars.

> ## Changes in 0.5.2 (in development)

Reviewer 2: ## Changes in 0.1.6 (in development)

> ## Changes in 0.5.2 (in development)

Reviewer 2: ```suggestion
## Changes in 0.1.6 (in development)
